Configuration/TCA/tx_seminars_seminars.php

```diff
@@ -128,8 +128,6 @@
             'config' => [
                 'type' => 'select',
                 'renderType' => 'selectMultipleSideBySide',
-                'internal_type' => 'db',
-                'allowed' => 'tx_seminars_seminars',
```

Comment:
Is this the only place where those properties can be removed, or are there also others which we can clean up as well?